### PR TITLE
Adding new frontend

### LINKED ISSUE #47: Create new App in Nuxt

### PR DESCRIPTION
**Pull request by K42IE**
Adding the new Nuxt repo to the existing codebase. 

fixes #47

### CODE REVIEW

**Review by RobertLRead — APPROVED** [review]
I've cast my eyes over this and it looks good to the extent that I understand it. I need to more fully understand how we are keeping the front end separate from the backend.

File: public/ThumbsRating.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Allows user to select a thumbs up or thumbs down rating to be stored with their entry.
+import html from './html.js';
```
Comment (devhawk) [inline public/ThumbsRating.js:2]: Why is this file JS instead of TS?

File: public/ThumbsRating.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
```
Comment (devhawk) [inline public/ThumbsRating.js:3]: Also, should this file be in the `/packages/frontend/public` folder instead of the `/public` folder?

**Review by devhawk — CHANGES_REQUESTED** [review]
requested changes in code comments

File: public/ThumbsRating.js (hunk 1 above)
Comment (devhawk) [reply public/ThumbsRating.js:2]: Maybe a better question is why isn't this a component?

**Review by devhawk — APPROVED** [review]

**Pull request merged**